kms.tf

```diff
@@ -76,6 +77,51 @@ resource "aws_kms_alias" "domino" {
 }
 
 data "aws_kms_key" "key" {
-  count  = var.use_kms ? 1 : 0
-  key_id = coalesce(var.kms_key_id, try(aws_kms_key.domino[0].arn, null))
+  count  = var.use_kms && var.kms_key_id != null ? 1 : 0
+  key_id = var.kms_key_id
+}
+
+## EKS key
+data "aws_iam_policy_document" "kms_key_eks" {
```

Comment:
eks_kms?

what does this have to do with eks?  seems like a generic policy for all kms mgmt.

Reply:
Yeah, I moved this back so it's clearer - for clusters where `use_kms` is false, I don't want to remove the existing KMS encryption for secrets.